builder-22/builder.toml

```diff
@@ -43,3 +47,8 @@ version = "0.14.1"
   [[order.group]]
     id = "heroku/java"
     version = "0.6.5"
+
+[[order]]
```

Comment:
Should the procfile buildpack be added to the Go group? (Similar to what I'll be doing for the Python buildpack)

For the other languages, they include the procfile buildpack in their meta-buildpack. However since the Go buildpack is not a meta-buildpack, the procfile buildpack isn't included. 

Whilst the Go buildpack does set default process types (as can be seen here: as can be seen here: https://github.com/heroku/builder/actions/runs/3623151336/jobs/6108735078#step:8:29) - if end users have a `Procfile` (for example to add additional commands used by their app, to customise args for the web process etc), then it will silently not do anything.

For example, the Go getting started guide actually has a `Procfile` at the moment (although it's very similar to the auto-detected processes, with the exception that without the Procfile, the Go builldpack actually configures two process, a `web` and a redundant `go-getting-started` - whereas the Procfile would have just configured `web`):
https://github.com/heroku/go-getting-started/blob/main/Procfile

Reply:
Yeah, I considered adding the procfile buildpack to this group, but wanted feedback from y'all. I was surprised we haven't already added an optional procfile buildpack for Node.js and Java already, so it felt odd to add it for only Go. Honestly, I'm in favor, and was considering a separate PR to add Procfile to all the groups in this builder.